src/main/java/de/retest/web/RecheckSeleniumAdapter.java

```diff
@@ -49,19 +50,36 @@ public RecheckSeleniumAdapter() {
 
 	@Override
 	public boolean canCheck( final Object toVerify ) {
-		return toVerify instanceof WebDriver;
+		return toVerify instanceof WebDriver || toVerify instanceof RemoteWebElement;
```

Comment:
What about `WrapsElement` and `WrapsDriver` to ensure the widest spectrum possible. 

See: [extract](https://github.com/retest/recheck-web/blob/6d724b7123aeb356d4180cefd35056ab9822512e/src/main/java/de/retest/web/adapter/WebElementRecheckAdapter.java#L52) and [execute](https://github.com/retest/recheck-web/blob/6d724b7123aeb356d4180cefd35056ab9822512e/src/main/java/de/retest/web/mapping/JavaScriptWebDataProvider.java#L115).

Reply:
Very good catch!